tests/UI/specs/EmptySite_spec.js

```diff
@@ -61,22 +61,6 @@ describe("EmptySite", function () {
     expect(await pageElement.screenshot()).to.matchImage('trackingCode');
   });
 
-  it('should show the advanced tracking options when clicked', async function () {
```

Comment:
I think we should keep the tests for the advanced options, are we not keeping the standard JS option in the no-data screen at all?

Reply:
@michalkleiner done :+1: